organizational/slack.rst

```diff
@@ -1,12 +1,12 @@
 HacSoc Slack
 ============
 
-`Hacsoc Slack`_ 
+`Click here to go to the official HacSoc Slack.`_ 
 
 .. _Hacsoc Slack: https://hacsoc.slack.com
```

Comment:
I'll need to update `push.sh` to check whether it is within a PR (just an environment variable check, NBD), but we can't switch it yet